### PR TITLE
add background color when input is not added and when click on submit

### PR DESCRIPTION
**Pull request by tannazma**
add validation form in the main branch

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -15,11 +15,6 @@ document.querySelector(".calculate").addEventListener("click", function (e) {
     const SH = document.querySelector("#sh")
     const AR = document.querySelector("#ar")
     const area = (RU.value / (PE.value * SH.value)) * AR.value;
-    console.log(RU.value)
-    console.log(PE.value)
-    console.log(SH.value)
-    console.log(AR.value)
-    console.log(area)
```
Comment (aghArdeshir) [inline script.js:22]: :+1:

File: script.js [hunk 2]
```diff
@@ -33,3 +28,12 @@ function validateNumber() {
     }
 
 }
+document.querySelector(".calculate").addEventListener("click", function () {
```
Comment (aghArdeshir) [inline script.js:31]: this makes the validation work only on `click` of the calculate button. But we want the same function to execute also upon submitting form by any other means (e.g. user press enter, etc...)

It is better to use `form.addEventListener('submit')`, which also includes clicking on calculate button

File: script.js [hunk 3]
```diff
@@ -33,3 +28,12 @@ function validateNumber() {
...
+    let inputFields = document.querySelectorAll("input");
```
Comment (aghArdeshir) [inline script.js:32]: best practice is to use `const` whenever possible vs. let

File: script.js [hunk 4]
```diff
@@ -33,3 +28,12 @@ function validateNumber() {
...
+    for (const inputField of inputFields) {
+        if (inputField.value=="") {
```
Comment (aghArdeshir) [inline script.js:34]: best practice is to avoid using `==`. Instead use `===`

Comment (aghArdeshir) [inline script.js:34]: Please fix formatting: spaces around `===`

File: styles.css [hunk 5]
```diff
@@ -75,3 +75,6 @@ input {
 .result {
   color: white;
 }
+.errorBox {
+  background-color: rgb(182, 103, 103);
```
Comment (aghArdeshir) [inline styles.css:79]: it is best practice to give names to your colors with css variables and use them here, e.g.:
```
:root {
  --error-color: rgb(182, 103, 103);
}
```

and then use it here:
```
.errorBox {
  background-color: var(--error-color);
}
```

**Review by aghArdeshir — CHANGES_REQUESTED** [review]

File: script.js (hunk 2 above)
Comment (tannazma) [reply script.js]: Done: https://github.com/tannazma/prework/pull/1/commits/d4825c871caa1021ed5c1075be6d886ec2ffa453

 thanks

File: script.js (hunk 3 above)
Comment (tannazma) [reply script.js]: Done: https://github.com/tannazma/prework/pull/1/commits/a0b311d38f5c618827c5f38184f1f2e57ecd6bfe

Thank you

File: script.js (hunk 4 above)
Comment (tannazma) [reply script.js]: Done already

File: styles.css (hunk 5 above)
Comment (tannazma) [reply styles.css]: Thanks for letting me know CSS root selector: https://github.com/tannazma/prework/pull/1/commits/228ddb0fd58dcdc313647e24b5c6d3046f2ef07c

File: script.js (hunk 4 above)
Comment (aghArdeshir) [reply script.js]: No it is not done

Comment (tannazma) [reply script.js]: https://github.com/tannazma/prework/pull/1/commits/9c4c22ed561bd1e992f8c5a287f92ac515a2b8cc Soryy now it's done

**Review by aghArdeshir — APPROVED** [review]
Looks great :rocket:

**Pull request merged**